### PR TITLE
Update zh-TW & ja locale with new v4 keys

### PR DESCRIPTION
**Pull request by hohayo**
Add translation for toggle menu

update_toggle_menu_translation

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -16,6 +16,16 @@ en:
...
+    toggle_section:
+      show: "Show"
+      hide: "Hide"
+    toggle_user_menu:
+      sign_out: "Sign Out"
+    toggle_dark_mode:
+      dark_mode: "Dark Mode"
+      light_mode: "Light Mode"
+    toggle_main_navigation_menu:
+      main_navigation: "Main Navigation"
```
Comment (tagliala) [inline config/locales/en.yml:28]: Hi, these are unwanted changes?

I do not expect changes in English locale file. If we need a change here, then we have a different bug.

Please find the following below:

```
    toggle_dark_mode: Toggle dark mode
    toggle_main_navigation_menu: Toggle main navigation menu
    toggle_section: Toggle section
    toggle_user_menu: Toggle user menu
```

You may want to run `i18n-tasks add-missing` to populate your locale of interest with missing locales, and then start from there

File: config/locales/zh-TW.yml [hunk 2]
```diff
@@ -52,6 +52,16 @@
...
+    toggle_dark_mode:
+      dark_mode: "暗黑模式"
+      light_mode: "明亮模式"
+    toggle_main_navigation_menu:
+      main_navigation: "主要導覽"
+    toggle_section:
+      show: "展開"
+      hide: "收合"
+    toggle_user_menu:
+      sign_out: "登出"
```
Comment (tagliala) [inline config/locales/zh-TW.yml:64]: @hohayo please translate the following:

```
    toggle_dark_mode: Toggle dark mode
    toggle_main_navigation_menu: Toggle main navigation menu
    toggle_section: Toggle section
    toggle_user_menu: Toggle user menu
```

**Review by tagliala — CHANGES_REQUESTED** [review]

File: config/locales/zh-TW.yml (hunk 2 above)
Comment (hohayo) [reply config/locales/zh-TW.yml]: @tagliala 
Resolved

**Review by tagliala — CHANGES_REQUESTED** [review]
Thanks,

please remove changes from `en.yml`

**Review by tagliala — APPROVED** [review]

**Pull request merged**